lib/improver/argparser.py

```diff
@@ -28,19 +28,185 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-"""Common option utilities for Improver CLIs."""
+"""Common option utilities for improver CLIs."""
 
-import argparse
+from argparse import ArgumentParser
 
 
-class ArgParser(argparse.ArgumentParser):
-    """Argument parser for improver CLIs"""
+class ArgParser(ArgumentParser):
+    """Argument parser for improver CLIs.
+    (Subclass of argparse.ArgumentParser)"""
+
+    # CENTRALIZED_ARGUMENTS, is a dictionary where, for each element:
```

Comment:
Please include all these comments in a single class docstring, and define the dictionaries below.